test-node.ts

```diff
@@ -0,0 +1,34 @@
+import { NodeConnectionManager } from './src/nodes';
```

Comment:
I think this file should be removed.

Reply:
Yep, this can be removed. Accidental commit by me - was just testing the `RWLock` with this.

Reply:
Removed